contrib/llvm-project-cmake/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ if (NOT ENABLE_EMBEDDED_COMPILER AND NOT ENABLE_DWARF_PARSER AND NOT ENABLE_BLAK
     return()
 endif()
 
-set (LLVM_VERSION "18.0.0bundled")
+set (LLVM_VERSION "19.0.0bundled")
```

Comment:
The LLVM submodule currently contains three commits ([here](https://github.com/ClickHouse/llvm-project/commits/ClickHouse/release/19.x/)). The commit history doesn't matter too much, could you squash these commits into a single commit and force-push to branch `ClickHouse/release/19.x`?

Reply:
Yes, will do, I didn't finish with reverting and documenting patches yet.